src/AES.java

```diff
@@ -0,0 +1,50 @@
+package src;
+
+import java.util.Base64;
+import javax.crypto.Cipher;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.IvParameterSpec;
+
+// This is a child of Encryption class 
+
+public class AES extends Encryption {
+
+    // Overwrites the Encrypt class to implement the encryption of AES
+
+    static byte[] cipherText;
+    static byte[] plainText;
```

Comment:
All attributes should be private or protected unless final.  These two attributes it may make sense to extract them to parent Encryption class